plain2json.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
```

Comment:
maybe call it txt2json ?

Reply:
I don't see a change in gitignore to ignore list.json

Reply:
could you please not reverse the commit platform/* files?

Reply:
> maybe call it txt2json

Sure.

> I don't see a change in gitignore to ignore list.json

Will do. But do we need list.json at all? I kept it in case somebody needs it, but it would be simpler to just fill the array from txt file and pass in to generator, without intermediary json file at all. What do you think?

> could you please not reverse the commit platform/* files?

Not sure I get the idea. I regenerated them with `npm run build`. Should I leave them unchanged instead, as they are in master branch?

Reply:
> but it would be simpler to just fill the array from txt file and pass in to generator, without intermediary json file at all. What do you think?


Indeed, but I don't have the time right now to implement this so it's better that way.
list.json was never in the public API thus must not be relied on. 
And If it was (^^) that's not my problem™ :p `list.txt` will be easier to manage and less error-prone :+1:

>  I regenerated them with npm run build. Should I leave them unchanged instead, as they are in master branch?

Exactly, please leave them unchanged :)

Reply:
Okay, thanks. I'll fix naming and revert platform files. Also I'll try to get rid of `list.json` and make `loader.js` load data from `list.txt`. I'll be back ツ

Reply:
- Removed 'convertion' step and changed `loader.js` to load domains from `list.txt` instead of `list.json`.
- Reverted platform files.